pytorch_pfn_extras/onnx/pfto_exporter/export.py

```diff
@@ -473,11 +477,15 @@ def symbolic_function(self, n: torch._C.Node) -> Optional[Callable]:
             # TODO(twata): Use repr(pyobj) in scope name or doc_string
             return cast(Callable, pyobj.symbolic)
         else:
+            domain = ""
             if ns == "prim":
-                op = f"prim_{op}"
-            if sym_reg.is_registered_op(op, "", self.opset_version):  # type: ignore[no-untyped-call]
+                if pytorch_pfn_extras.requires('1.11'):
+                    domain = "prim"
+                else:
+                    op = f"prim_{op}"
+            if sym_reg.is_registered_op(op, domain, self.opset_version):  # type: ignore[no-untyped-call]
                 return cast(
-                    Callable, sym_reg.get_registered_op(op, "", self.opset_version)  # type: ignore[no-untyped-call]
+                    Callable, sym_reg.get_registered_op(op, domain, self.opset_version)  # type: ignore[no-untyped-call]
```

Comment:
This fix is for the structural changes of sym_reg._registry between torch<1.11 and torch>=1.11. It can be confirmed with the following script.

```py
import torch.onnx.symbolic_registry as sym_reg

sym_reg.register_version('', 9)
sym_reg.register_version('', 10)
sym_reg.register_version('', 11)
sym_reg.register_version('', 12)

for k, v in sym_reg._registry.items():
    print(k)
    for v1, v2 in v.items():
        print(f'  {v1}')
```